tools/DocsLinter/Tests/LinkCheckingTests.cs

```diff
@@ -21,8 +21,9 @@ private static RemoteLink GetUrlForStatusCode(int statusCode)
         }
 
         [OneTimeSetUp]
-        public void IgnoreSslErrors() {
-            System.Net.ServicePointManager.ServerCertificateValidationCallback += 
+        public void IgnoreSslErrors()
```

Comment:
Could these drive-by formats (and the s->z rename) be split into a different PR?

Reply:
Yeah, that was the linter's doing. I will revert the drive-by changes.